src/main/java/lesson4/ApacheWorker.java

```diff
@@ -0,0 +1,66 @@
+package lesson4;
+
+import org.apache.http.*;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.impl.client.HttpClientBuilder;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ApacheWorker {
+
+    public static List<String> getListOfRepoUsers() throws Exception {
```

Comment:
getListOfRepoUsers method do 2 things: taking care of the way how to send the request and parse the output. Try to keep methods targeted to one goal: either prepare request or analyze the response

Reply:
Splitted the method into two methods